apps/emqx_bridge_mqtt/src/emqx_bridge_rpc.erl

```diff
@@ -61,7 +61,7 @@ stop(#{client_pid := Pid}) when is_pid(Pid) ->
     ok.
 
 %% @doc Callback for `emqx_bridge_connect' behaviour
--spec send(node(), batch()) -> {ok, ack_ref()} | {error, any()}.
+-spec send(#{address:=atom() | {_,atom()}, _=>_}, batch()) -> {ok, ack_ref()} | {error, any()}.
```

Comment:
the address field's spec should be like this:
https://github.com/priestjim/gen_rpc/blob/master/include/types.hrl#L8